presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -135,6 +135,7 @@
     public static final String PUSH_PARTIAL_AGGREGATION_THROUGH_JOIN = "push_partial_aggregation_through_join";
     public static final String PARSE_DECIMAL_LITERALS_AS_DOUBLE = "parse_decimal_literals_as_double";
     public static final String FORCE_SINGLE_NODE_OUTPUT = "force_single_node_output";
+    public static final String FORCE_COORDINATOR_EXECUTION = "force_coordinator_execution";
```

Comment:
Maybe something like `force_singe_node_execution_for_values_only_queries` (though it seems to be verbose).

@aweisberg Can you think of any better name?

Reply:
how about this name `FORCE_COORDINATOR_NODE_EXECUTION_FOR_VALUES_QUERIES`